assets/styles/components/organisms/SfMegaMenu.scss

```diff
@@ -0,0 +1,75 @@
+.sf-mega-menu {
+  box-shadow: 0 2px 4px 0 var(--_c-gray-shadow);
+
+  &__menu {
+    justify-content: flex-start;
+  }
+
+  &__content {
+    z-index: 1;
+    position: relative;
+    padding: 0 3%;
+    max-width: var(--mega-menu-content-width, 100%);
+  }
+}
+
+.sf-mega-menu-column {
+  flex: none;
+  padding-right: 5rem;
+
+  @include for-desktop {
+    &__header {
+      border-bottom: 0.25rem var(--c-white) solid;
+      text-transform: capitalize;
+      font-size: 17px;
+      margin: 9px 0 5px 0;
+    }
+    &__content {
+      display: none;
+      position: absolute;
+      top: auto;
+      align-items: center;
+      padding: 0 2%;
+      background-color: var(--c-white);
+      box-shadow: 0 2px 4px 0 var(--_c-gray-shadow);
+      width: 92vw;
```

Comment:
Avoid using pixel, better to use either spacers or em every were

Reply:
used `rem` instead `px`